app/frontend/src/components/forms/submission/ManageSubmissionUsers.vue

```diff
@@ -0,0 +1,303 @@
+            <v-simple-table dense>
+              <template v-slot:default>
+                <thead>
+                  <tr>
+                    <th class="text-left">Name</th>
+                    <th class="text-left">Username</th>
+                    <th class="text-left">Email</th>
+                    <th class="text-left" v-if="isDraft">Actions</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr :key="item.userId" v-for="item in userTableList">
+                    <td>{{ item.fullName }}</td>
+                    <td>{{ item.username }}</td>
+                    <td>{{ item.email }}</td>
+                    <td v-if="isDraft">
+                      <v-btn
+                        color="red"
+                        icon
+                        :disabled="item.isOwner"
+                        @click="removeUser(item)"
+                      >
+                        <v-icon>remove_circle</v-icon>
+                      </v-btn>
+                    </td>
+                  </tr>
+                </tbody>
+              </template>
+            </v-simple-table>
+          </v-skeleton-loader>
+        </v-card-text>
+
+        <v-card-actions class="justify-center">
+          <v-btn class="mb-5 close-dlg" color="primary" @click="dialog = false">
+            <span>Close</span>
+          </v-btn>
+        </v-card-actions>
+      </v-card>
+
+      <BaseDialog
+        v-model="showDeleteDialog"
+        type="CONTINUE"
+        @close-dialog="showDeleteDialog = false"
+        @continue-dialog="modifyPermissions(userToDelete.id, []); showDeleteDialog = false"
+      >
+        <template #title>Remove {{ userToDelete.username }}</template>
+        <template #text>
+          Are you sure you wish to remove
+          <strong>{{ userToDelete.username }}</strong
+          >? They will no longer have permissions for this submission.
+        </template>
+        <template #button-text-continue>
+          <span>Remove</span>
+        </template>
+      </BaseDialog>
+    </v-dialog>
+  </span>
+</template>
+
+<script>
+import { mapActions } from 'vuex';
+
+import { FormPermissions } from '@/utils/constants';
+import { rbacService, userService } from '@/services';
+
+export default {
+  name: 'ManageSubmissionUsers',
+  props: {
+    isDraft: {
+      type: Boolean,
+      required: true,
+    },
+    submissionId: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      dialog: false,
+      isLoadingTable: true,
+      showDeleteDialog: false,
+      userTableList: [],
+      userToDelete: {},
+
+      // search box
+      findUsers: null,
+      isLoadingDropdown: false,
+      userSearchResults: [],
+      userSearchSelection: null,
+    };
+  },
+  methods: {
+    ...mapActions('notifications', ['addNotification']),
+    // show users in dropdown that have a text match on multiple properties
+    async addUser() {
```

Comment:
can we remove the async since we are not awaiting anything?

Reply:
Indeed

Reply:
Fixed